common/lib/capa/capa/safe_exec/helpers.py

```diff
@@ -0,0 +1,92 @@
+"""
+Helper methods related to safe exec.
```

Comment:
@ericfab179 could we call this file `remote` or `remote_exec` to give a better idea of what kinds of helpers we should expect here?